spec/features/admin_spec.rb

```diff
@@ -36,5 +36,20 @@
     expect(page).to have_content("Leo")
   end
 
-
+  it "can add a new series" do
+    visit admin_index_path
+    page.click_link('Add a new series')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.